restli-int-test-api/src/main/pegasus/com/linkedin/restli/examples/defaults/api/FooDefault.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.restli.examples.defaults.api
+
+import com.linkedin.restli.common.CollectionMetadata
+
+record FooDefault {
```

Comment:
Do we need to add a new record for this use case? We already have a similar record called `LowLevelRecordWithDefault` in the package `com.linkedin.restli.examples.defaults.api. Could we reuse that instead?

Reply:
Yes, I revise this new schema to use the old one